doc/design/dist/submit-job.md

```diff
@@ -0,0 +1,161 @@
+
+You should configure PaddlePaddle client by the configuration file firstly, the default path:
+`$HOME/.paddle/config`.
+
+```yaml
+apiVersion: v1
+dockerRegistry:
+  domain: domain.com //default is docker.io
+  username: <username>
+  password: <password>
+paddleServer: http://<paddle server domain>:<paddle server port>
+```
+
+
+## Submit a distributed training job
+Users will submit a distributed training job with the command: `paddle train` without flag `--locally`.
+
+```bash
+paddle train \
+  --job-name=cluster-quickstart \
+  --package-path=$PWD/quick_start \
+  --module=quick_start.train \
+  --input=<input-dir> \
+  --output=<output-dir> \
+  --trainers=4 \
+  --pservers=2 \
+  --base-image:<paddle-k8s-image> \
+  --use-gpu=1 \
+  --gpu-num=1 \
+  --env="NUM_PASS=5"
+```
+
+- `job-name`: you should specify a unique job name
+- `package-path`: python package files on your host
+- `module`: include the main function, trainer entrance
+- `input`: input directory on distributed file system
+- `output`: output directory on distributed file system
+- `trainers`: trainer process count
+- `pserver`: parameter process count
+- `base-image`: your trainer docker image, include your trainer files and dependencies.
+- `use-gpu`: whether it is a GPU train
+- `gpu-num`: how much GPU card for one paddle trainer process, it's requirements only if `use_gpu=1`,
+- `env`: environment variable
+
+## Package into a docker image
+
+- `Runtime docker image` and `base docker image`
+
+  For a distributed training job, there is two docker image called `runtime docker image` and `base docker image`, the `runtime docker image` is actually running in kubernetes.
+
+  - `runtime docker image` include user's package files and all dependencies.
+  - `base docker image` usually is PaddlePaddle product docker image including paddle binary files and some scripts used for starting up the trainer process and fetch some information of pod. And of course, users can also build their own paddle binary files into the custom `base docker image` with [this doc](../../../paddle/scripts/docker/README.md).
+
+  `runtime docker image` will be built by PaddlePaddle client automatic, here is a simple example project:
+  ```bash
+  paddle_example
+    |-quick_start
+      |-trainer.py
+      `-dataset.py
+    requirments.txt
+  ```
+  - `requirements.txt` list python dependencies package, you can create is like:
+
+    ```txt
+    pillow
+    protobuf==3.1.0
+    ```
+    some other details is [here](https://pip.readthedocs.io/en/1.1/requirements.html).
+  - `quick_start` directory include the trainer package files.
+
+  Execute the command: `paddle train...`, PaddlePaddle client will upload the trainer package files and setup parameters to [Paddle Server](#paddle-server).
+
+## Paddle Server
+Paddle server is running on kubernetes, users will configure the server address in [PaddlePaddle client configuration file](#configurate-paddlepaddle-client)
+
+- HTTP server
```

Comment:
HTTP server -> Paddle Server

Btw, title need to have title case: HTTP Server, not HTTP server. You can convert a string to title case at: http://www.titlecase.com/